### ISSUE TITLE
add ability to creation new partitions to BLOM

### ISSUE THREAD

**Issue by mvertens** [NorESMhub/BLOM#279]
It would be really useful to have the ability to create new partitions for BLOM as part of the BLOM code.
This would include documentation and a user-friendly build. Its not clear to me what the process is for getting new partitions made and tested. This would be really helpful both for regression testing and for running BLOM on new grids and/or platforms.

**Comment by matsbn** [comment]
I agree it would be good to have this functionality and documentation available in the repository. The grid partitioning process involves some old fortran codes with a bit of trial and error to arrive at a useful partition. The code part should be suitable to rewrite into a scripting language, such as Python.
